privilege-management.md

```diff
@@ -154,7 +225,9 @@ You can use the `SHOW GRANTS` statement to see what privileges are granted to a
 
 ```sql
 SHOW GRANTS; -- show grants for the current user
+```
 
+```
```

Comment:
L236:

````markdown
```sql
SHOW GRANTS FOR 'root'@'%'; -- show grants for a specific user
```
````

Reply:
9a0451782892a9e699cc74029973783658b59e48